napalm/__init__.py

```diff
@@ -26,6 +27,7 @@ def get_network_driver(vendor):
         'JUNOS': JunOSDriver,
         'JUNIPER': JunOSDriver,
         'FORTIOS': FortiOSDriver,
+#        'IBM': IBMDriver,
```

Comment:
I'll try to create pip package for the Lenovo/IBM library used in my module. After that we can easily add it to requirements file. I'll also prepare documentation. It should be ready soon ~around weekend :)